### DISCUSSION TITLE
Is there a way to interrupt long-running jobs without having to stop A1111?

### DISCUSSION THREAD

**Discussion by sdUserzz** [AIrjen/OneButtonPrompt#95] [Q&A] [answered]
Hi,
First off, I've been really getting a kick out of using this script!  It's taken my interest and productivity with SD to new levels.
One thing I can't seem to find...maybe I'm just missing something:
Is there a way to stop a job initiated by this script?  
I've found myself a number of times in the middle of a multi-looped upsize and after seeing the first iteration of the image I know I'm not going to be using it.  The rest of the job is just a waste of time at that point.
I realize I can just kill the A111 processes, but I don't believe that's the best idea generally and I don't want to have to re-enter all the parameters I've set.

**Comment by AIrjen** [comment, marked as answer]
Hi, you can use the normal generation and img2img as well with this script. I know sometimes people miss this, especially after Olivio's video.

For One Button Run and Upscale, there is currently no "interrupt" button.

You can also run normally for a while, and then use the "only upscale" option in One Button Run and Upscale.

There is also the Quality Gate options you can look into, which tries to determine wether a image is of a good enough quality to continue.

**Comment by AIrjen** [comment]
Hi!I hacked in a interrupt button. Its not perfect, it only interrupts what it is currently doing. So it doesn't STOP the process. But it should help with with what you want, and that is to skip a current image!

![image](https://github.com/AIrjen/OneButtonPrompt/assets/130234949/02806bf3-f4ee-429c-9ac2-445ab299a414)
